scripts/build-espresso.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const { logger, fs } = require('@appium/support');
+const { ServerBuilder } = require('../build/lib/server-builder.js');
+
+const LOG = new logger.getLogger('EspressoBuild');
+
+const ROOT_DIR = path.resolve(__dirname, '..');
+const ESPRESSO_SERVER_ROOT = path.join(ROOT_DIR, 'espresso-server');
+
+const ESPRESSO_SERVER_BUILD = path.join(ESPRESSO_SERVER_ROOT, 'app', 'build');
+
+async function buildEspressoServer () {
+  console.log(`Deleting the build directory ${ESPRESSO_SERVER_BUILD}`); // eslint-disable-line no-console
+
+  const opts = {
+    serverPath: ESPRESSO_SERVER_ROOT,
+    showGradleLog: process.env.SHOW_GRADLE_LOG ? process.env.SHOW_GRADLE_LOG : false
+  };
+
+  if (process.env.TEST_APP_PACKAGE) {
+    opts.testAppPackage = process.env.TEST_APP_PACKAGE;
+  }
+
+  if (process.env.ESPRESSO_BUILD_CONFIG) {
+    if (!(await fs.exists(process.env.ESPRESSO_BUILD_CONFIG))) {
+      throw Error(`'${process.env.ESPRESSO_BUILD_CONFIG}' did not exist. Please set the path as an absolute path.`);
+    }
+    try {
+      const buildConfigurationStr = await fs.readFile(process.env.ESPRESSO_BUILD_CONFIG, 'utf8');
+      opts.buildConfiguration = JSON.parse(buildConfigurationStr);
+      console.log(`The espresso build config is ${JSON.stringify(opts.buildConfiguration)}`); // eslint-disable-line no-console
```

Comment:
why cannot we just use the logger above to print it?

Reply:
same above

Reply:
👍 [56b2d76](https://github.com/appium/appium-espresso-driver/pull/858/commits/56b2d768920a4e995b2d660a6df80c955b8a3327)